sympy/physics/mechanics/system.py

```diff
@@ -0,0 +1,437 @@
+        combined equations of motion. The presence of these constraints requires
+        that a DAE solver be used instead of an ODE solver. If the system is
+        given in form [3] the alg_con variable will be adjusted such that it is
+        a representation of the combined kinematics and dynamics thus make sure
+        it always matches the mass matrix entered.
+
+    dyn_implicit_mat : Matrix
+        This is the M matrix in form [3] of the equations of motion (the mass
+        matrix or generalized inertia matrix of the dynamical equations of
+        motion in implicit form).
+
+    dyn_implicit_rhs : Matrix
+        This is the F vector in form [3] of the equations of motion (the right
+        hand side of the dynamical equations of motion in implicit form).
+
+    comb_implicit_mat : Matrix
+        This is the M matrix in form [2] of the equations of motion. This matrix
+        contains a block diagonal structure where the top left block (the first
+        rows) represent the matrix in the implicit form of the kinematical
+        equations and the bottom right block (the last rows) represent the
+        matrix in the implicit form of the dynamical equations.
+
+    comb_implicit_rhs : Matrix
+        This is the F vector in form [2] of the equations of motion. The top
+        part of the vector represents the right hand side of the implicit form
+        of the kinemaical equations and the bottom of the vector represents the
+        right hand side of the implicit form of the dynamical equations of
+        motion.
+
+    comb_explicit_rhs : Matrix
+        This vector represents the right hand side of the combined equations of
+        motion in explicit form (form [1] from above).
+
+    kin_explicit_rhs : Matrix
+        This is the right hand side of the explicit form of the kinematical
+        equations of motion as can be seen in form [3] (the G matrix).
+
+    dynamic_symbols : Matrix
+        This matrix contains all of the symbols that have a time dependence in
+        the equations of motion of the system
+
+    constant_symbols : Matrix
+        This matrix contains all of the symbols that do not have a time
+        dependence in the equations of motion of the system
+
+    output_eqns : Dictionary
+        If output equations were given they are stored in a dictionary where the
+        key corresponds to the name given for the specific equation and the
+        value is the equation itself in symbolic form
+
+    bodies : List
+        If the bodies in the system were given they are stored in a list for
+        future access
+
+    loads : List
+        If the loads in the system were given they are stored in a list for
+        future access. This includes forces and torques where forces are given
+        by (point of application, force vector) and torques are given by
+        (reference frame acted upon, torque vector).
+
+    Parameters
+    ==========
```

Comment:
Ah, there is a setting that you can list in sphinx to get the autodoc to grab this:

http://stackoverflow.com/questions/5599254/how-to-use-sphinxs-autodoc-to-document-a-classs-init-self-method